lib/graphql/schema.ts

```diff
@@ -2326,25 +2318,24 @@ export type InvoiceCustomerInput = {
 
 export type InvoiceOutput = {
   __typename?: 'InvoiceOutput';
-  id?: Maybe<Scalars['String']>;
   invoiceSettingsId?: Maybe<Scalars['String']>;
   customerId?: Maybe<Scalars['String']>;
-  customer?: Maybe<InvoiceCustomerOutput>;
   status: Scalars['String'];
-  invoiceNumber?: Maybe<Scalars['Float']>;
   dueDate?: Maybe<Scalars['DateTime']>;
   note?: Maybe<Scalars['String']>;
+  id?: Maybe<Scalars['String']>;
+  customer?: Maybe<Customer>;
+  invoiceNumber?: Maybe<Scalars['Float']>;
   products?: Maybe<Array<InvoiceProductOutput>>;
 };
 
 export type InvoiceInput = {
-  id?: Maybe<Scalars['String']>;
   invoiceSettingsId?: Maybe<Scalars['String']>;
   customerId?: Maybe<Scalars['String']>;
-  status?: Maybe<Scalars['String']>;
-  invoiceNumber?: Maybe<Scalars['Float']>;
+  status: Scalars['String'];
```

Comment:
- Status is now mandatory, since we implemented the DRAFT functionality. 
- invoiceNumber is not needed in the `invoiceInput` because it is generated in the BE response when we're finalizing an invoice ; so it is only needed in the `InvoiceOutput`